code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -269,6 +269,11 @@ AI MODULES
 	desc = "A 'reset' AI module: 'Clears all, except the inherent, laws.'"
 	origin_tech = list(TECH_DATA = 3, TECH_MATERIAL = 4)
 
+// VOREstation edit: use map default laws
+/obj/item/weapon/aiModule/reset/Initialize()
+	. = ..()
+	laws = new global.using_map.default_law_type // Pull from loaded map
```

Comment:
What happens if there is no default law defined on the map?

Reply:
Default is var/default_law_type in /maps/~map_system/maps.dm. This will break if that's changed to null for some reason or a child map datum overrides it incorrectly.